tests/src/baseservices/threading/waithandle/waitany/nullarray.cs

```diff
@@ -31,6 +31,8 @@ static int Main()
 			retCode = 98;			
 		}
 
+		GC.KeepAlive(waitHandles);
```

Comment:
I agree.  What was the observed behavior of this test under GCStress without this change?

---
In reply to: [132806971](https://github.com/dotnet/coreclr/pull/13351#discussion_r132806971) [](ancestors = 132806971)